.github/workflows/build.yml

```diff
@@ -29,20 +29,19 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    environment: production
-    env:
-      NODE_VER: 16.14.0
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
I like having matrix here to be able to build with multiple node versions.
Do we want to add any other node version here?

Reply:
Yes Deepanshu, I agree, added this to track https://github.com/dapr/js-sdk/issues/494